app/Http/Controllers/InventoryController.php

```diff
@@ -175,7 +175,15 @@ public function update(Request $request, Inventory $inventory)
      */
     public function destroy(Inventory $inventory)
     {
-        //
+        try
+        {
+            $inventory->delete();
+            return redirect('/inventory')->with('success', 'Successfully deleted item.');
+        }
+        catch(\Exception $e)
+        {
+            return redirect('/inventory')->with('error', 'Error deleting item.');
```

Comment:
Change `error` message to `Can't delete item. This item already exists in Receipts/Bills`.